docs/modules/user-guide/partials/proc_refering-to-a-parent-devfile-in-a-devfile.adoc

```diff
@@ -2,18 +2,22 @@
 = Refering to a parent devfile in a devfile
 
 [role="_abstract"]
-It is possible to refer to another devfile as a parent to a given devfile. If a parent is specified, the devfile will inherit all behaviour from the parent. Part of the content of the parent can be overridden in the child devfile. That allows refering a devfile (the parent) from a distinct devfile and makes it possible to reuse in multiple devfiles the same parent (the stack).
-
 .Prerequisites
 
 * xref:adding-schema-version-to-a-devfile.adoc[]
 * xref:adding-a-name-to-a-devfile.adoc[]
 
 
+This section teaches you how to specify a parent devfile. You can choose a devfile and refer to another devfile as its parent. If you specify a parent, the child devfile inherits all its behavior from its parent devfile. Still, you can use the child devfile to override certain content from the parent devfile. If you override the correct content, you can reuse the same parent devfile in multiple other devfiles. If you do reuse a parent devfile, the parent turns into a stack that is used in multiple other devfiles.
+
 .Procedure
 
-* A parent devfile can be referenced in 3 different ways: `registry`, `uri` or `kubernetes`.
-+
+You can refer to a parent devfile in three different ways:
+
+. registry
```

Comment:
don't number a list if it is not a fixed order of steps that need to be taken. Use an unordered list.

Reply:
also, the points are rather short and unspecific. What kind of registry, for example? A minimum amount of information should be given.